server/src/weaverbird/pipeline/steps/date_extract.py

```diff
@@ -23,20 +23,24 @@
     "milliseconds",
 ]
 
+FIRST_DAY_DATE_PARTS = Literal[
```

Comment:
This name might be misleading as it also contains `previousDay`. Also, the name should be and indication about why it is needed. From what I understand, these should be timestamp types in the end, right ? how about distinguishing between `TIMESTAMP_DATE_PARTS` and `INTEGER_DATE_PARTS` ?

Reply:
Good idea 👌🏽